rules/framework.bzl

```diff
@@ -298,7 +314,6 @@ def _apple_framework_packaging_impl(ctx):
         "multi_arch_linked_archives",
         "multi_arch_linked_binaries",
         "multi_arch_dynamic_libraries",
-        "source",
```

Comment:
Do we need source when not virtualize_frameworks?

Reply:
Probably can revert this